general/releases.md

```diff
@@ -75,17 +75,23 @@ The most recent [long-term support release (LTS)](https://en.wikipedia.org/wiki/
 
 These are the target dates for releases. These dates may vary slightly due to unforeseen circumstances.
 
-| Release type | Frequency | Months |
+| Release type | Frequency | Months | Timing |
 | --- | --- | --- |
-| [Major](./development/process.md#major-release-cycles) (eg. 3.x) | 6 monthly | Second Monday of May and November |
-| [Minor](./development/process.md#stable-maintenance-cycles) (Point) (eg. 3.x.y) | 2 monthly | Second Monday of July, September, November, March and May. Third Monday of January. |
+| [Major](./development/process.md#major-release-cycles) (eg. 3.x) | 6 monthly | April, and October | Weeks 17, and 43 of the year |
```

Comment:
I may have missed it, but instead of the week number (or in addition to it), it may be easier to understand if we mention something like every 2nd Monday.

Reply:
It isn't every 2nd monday.
It's every 26 weeks, which is no longer every 2nd monday :(